mxcube3/routes/SampleCentring.py

```diff
@@ -601,8 +601,29 @@ def accept_centring():
 
 @mxcube.route("/mxcube/api/v0.1/sampleview/centring/reject", methods=['PUT'])
 def reject_centring():
-    """
-    Reject the centring position.
-    """
+    """Reject the centring position."""
     mxcube.diffractometer.rejectCentring()
     return Response(status=200)
+
+
+@mxcube.route("/mxcube/api/v0.1/sampleview/moveto", methods=['PUT'])
+def pos_from_x_y():
```

Comment:
It seems to me this method is to move a (x, y) position on sample video to beam position. Right ?

There are a number of problems:
- the beam might not be at the center (it is the case for MD2M, MD2, MD2S, MD3, ...) but believe me it can be elsewhere (on PX1 at Soleil maybe... Or any other diffractometer that doesn't assume beam on the centre of sample video)
- this assumes (x, y) positions from video corresponds to phiy and phiz roles, which we cannot know for all diffractometers

In fact there is a `moveToBeam` method (in MiniDiff.py) or `start_move_to_beam` (in GenericDiffractometer.py) that should be used, to delegate all the job to the Hardware Object not the Python server.